documentation/modules/exploit/linux/http/hadoop_unauth_exec.md

```diff
@@ -0,0 +1,112 @@
+## Description  
+
+This module exploits an unauthorized command execution vulneralbility in Apache Hadoop through ResourceManager REST API.
```

Comment:
Suggestion. Semantics.

`unauthorized` is kind of implied.

`unauthenticated` appears more applicable and also matches with the module file name `hadoop_unauth_exec`. Although, it's possible I misunderstand the issue, as a quick read of the references reveals `unauthorized` takes on a somewhat different meaning in the Hadoop cluster security model context.

Either way, the module file name should match the module name.

Reply:
typo: `vulnerability`